carto/bq.py

```diff
@@ -0,0 +1,305 @@
+import os
+import time
+import requests
+from .utils import ResponseStream
+from .exceptions import CartoException
+
+VALID_TYPES = [
+    'STRING', 'BYTES', 'INTEGER', 'INT64', 'FLOAT',
+    'FLOAT64', 'BOOLEAN', 'BOOL', 'TIMESTAMP', 'DATE', 'TIME',
+    'DATETIME', 'GEOMETRY'
+]
+
+TYPES_MAPPING = {
+    'GEOMETRY': 'GEOGRAPHY'
+}
+
+DATASETS_BASE_PATH = 'api/v4/data/observatory/bq/datasets'
```

Comment:
these paths have to change, see https://github.com/CartoDB/data-observatory/pull/694